src/singletons/Toasts.hpp

```diff
@@ -7,17 +7,29 @@ namespace chatterino {
 
 enum class Platform : uint8_t;
 
+enum class ToastReactions {
```

Comment:
The casing(?) of the enums don't match the rest of the codebase, we use PascalCase, not camelCase
see https://github.com/fourtf/chatterino2/blob/3f41dfeff942d9becb311a4b1769e8445da016af/src/messages/layouts/MessageLayout.hpp#L24-L29